tests/test_integration_patterns.py

```
@@ -396,6 +396,92 @@ async def test_api_integration_chain(self, mock_config, mock_client):
+            event.sender = "@user:matrix.org"
+            event.server_timestamp = 1234567890000
+
+            room = MagicMock()
+            room.room_id = "!room1:matrix.org"
+
+            await bot.on_room_message(room, event)
+
+            # Verify Bible text was fetched with correct parameters
+            mock_get_bible.assert_called_once()
+            call_args = mock_get_bible.call_args
+            assert "John 3:16" in call_args[0][0]  # passage argument
```

Ruff (0.12.2)

Use of `assert` detected

(S101)